### PR TITLE
C# Basic LINQ

### PR DESCRIPTION
**Pull request by Menighin**

### CODE REVIEW

File: src/pages/csharp/linq/first-or-default/index.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: FirstOrDefault
```
Comment (1andee) [inline src/pages/csharp/linq/first-or-default/index.md]: This should be `First Or Default` (with spaces)

File: src/pages/csharp/linq/to-dictionary/index.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+title: ToDictionary
```
Comment (1andee) [inline src/pages/csharp/linq/to-dictionary/index.md]: This should be named `To Dictionary`

File: src/pages/csharp/linq/to-list/index.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+title: ToList
```
Comment (1andee) [inline src/pages/csharp/linq/to-list/index.md]: This should be `To List`

**Comment by 1andee** [comment]
This looks good to me! Thanks so much for your contribution, @Menighin!

:ship:

**Comment by Menighin** [comment]
Glad to contribute! :D

**Pull request merged**